### PR TITLE
Fixed some clippy warnings

### PR DESCRIPTION
**Pull request by trevarj**
Others are false positives

### CODE REVIEW

File: crates/libtiny_client/src/state.rs [hunk 1]
```diff
@@ -719,16 +719,14 @@ async fn retry_channel_join(
...
 
-    match timeout(Duration::from_secs(10), rcv_abort.next()).await {
-        Err(_) => {
-            // Send join message
-            snd_irc_msg
-                .try_send(wire::join(std::iter::once(channel.as_ref())))
-                .unwrap();
-        }
-        Ok(_) => {
-            // Channel tab was closed
```
Comment (osa1) [inline crates/libtiny_client/src/state.rs:730]: Could you bring back this comment please? Maybe use put it before the `if`, something like

```
// Send join message after timeout. Error means channel tab was closed.
if timeout(...) { ... }
```

**Review by osa1 — APPROVED** [review]
Looks good, thanks @trevarj ! Left one minor comment.

File: crates/libtiny_client/src/state.rs (hunk 1 above)
Comment (trevarj) [reply crates/libtiny_client/src/state.rs:730]: Done. I think `Ok` means it was closed, right?

Comment (osa1) [reply crates/libtiny_client/src/state.rs:730]: Right, thanks!

**Pull request merged**